slave.mk

```diff
@@ -24,10 +24,18 @@ SRC_PATH = src
 RULES_PATH = rules
 TARGET_PATH = target
 DOCKERS_PATH = dockers
+ifdef BLDENV
```

Comment:
>BLDENV [](start = 6, length = 6)

We should assume BLDENV always defined. either STRETCH or JESSIE, and SID in future.

If keep backward compatibility, we could create target/debs as symlink to target/debs/jessie

Reply:
Actually backward compatibility should not be an issue. It is easy to fix build script or jenkins script. If this is true, we don't need a symlink.

---
In reply to: [275602303](https://github.com/Azure/sonic-buildimage/pull/2519#discussion_r275602303) [](ancestors = 275602303)